src/pages/instances/InstanceIps.tsx

```diff
@@ -12,9 +12,13 @@ const InstanceIps: FC<Props> = ({ instance, family }) => {
   const addresses = getIpAddresses(instance, family);
   return addresses.length ? (
     <ExpandableList
-      items={addresses.map((address) => (
-        <div key={address} className="ip u-truncate" title={address}>
-          {address}
+      items={addresses.map((val) => (
```

Comment:
Hm, not really convinced by `val`. I'd like `item` better, but it's not too important.